### ISSUE TITLE
Cosmos DB all versions and delete change feed bindings support

### ISSUE THREAD

**Issue by FinOCE** [Azure/azure-functions-dotnet-worker#2575]
### Description

All modes change feed support appears to have gone into public preview over a year ago, however you can still only use the latest version change feed in azure functions bindings. Are there plans for this to be made available? Has this change feed mode had progress made on it since this announcement? I am working on a project that would greatly benefit from this change feed support but it appears .NET only supports a pull model for it, which doesn't look appropriate for a function app anyway. https://azure.microsoft.com/en-au/updates/public-preview-azure-cosmos-db-all-versions-and-deletes-change-feed-mode/

**Comment by pglow11** [comment]
Yes, it would be great to know if this is coming up anytime soon.
